library/Jira/MonitoringInfo.php

```diff
@@ -164,6 +165,10 @@ public function getLastStateChange()
             return DateFormatter::formatDateTime($this->object->last_state_change);
         }
 
+        if ($this->object->state->last_state_change instanceof DateTime) {
+            return DateFormatter::formatDateTime($this->object->state->last_state_change->getTimestamp());
+        }
+
         return DateFormatter::formatDateTime($this->object->state->last_state_change);
```

Comment:
Please add a comment here, so telling to remove this once Icinga DB Web 1.1 is required.